### DISCUSSION TITLE
Use case of pockethost + Blazor WebAssembly + Azure static web app

### DISCUSSION THREAD

**Discussion by omarceloribeiro** [pockethost/pockethost#384] [Show and tell]
original project
https://github.com/neozhu/BlazorPocketApp

blazor webassembly standalone template connected to a  pockethost.io instance.

published to a Azure Static Web App.

mud blazor template
https://zealous-ocean-019afa910-preview.centralus.5.azurestaticapps.net/

default template, pointing to the same pockethost instance
https://happy-moss-0bb0d6110-preview.centralus.5.azurestaticapps.net/

only one parameter configuration:
on appsettings.json, "PocketbaseUrl": "https://blazorwasm-app1-api.pockethost.io/"

**Comment by benallfree** [comment]
Hello, I'm not sure what this means, can you please say more?

**Reply by omarceloribeiro** [reply]
sorry, the text was totally confused because it was a comment on another thread. sorry. now I adjusted it.

following additional explanation the core value:

pockethost makes possible to have a backend ready to go, to deliver a full real world application with zero costs and complexity. That is because exists the Blazor webAssembly, that is a full application framework that runs 100% on client based only in static files, (and it runs c# instead of javascript), also the Visual Studio provides a start template for this Blazor WebAssembly that is a full SPA app, with pages, menu, etc. the only thing need is a backend, that your pockethost solves gracefully.
then to host these static files of the blazor spa, the MS Azure offers a Static Web App (similar to vercel, for react) that is just a static file host, 100% free, and it also offers the option to configure a custom domain with no additional costs.

so your pockethost makes possible for a developer deliver a full app just by:
- creating the pockethost instance
- create the new project on VS with authenticstion and user data obtained from pocketbase collections
- configure the api url to pockethost instance
- click on publish to deploy it on azure
- and its done! just start adding new CRUDS and functionality

that is amazing!

the result it this project
https://happy-moss-0bb0d6110-preview.centralus.5.azurestaticapps.net/

**Reply by benallfree** [reply]
I see! Thank you for sharing :)
